str0m/src/media/register.rs

```diff
@@ -399,6 +480,21 @@ impl TimePoint {
     }
 }
 
+/// The max number of NACKs we perform for a single packet:
+const MAX_NACKS: u8 = 5;
```

Comment:
Open to debate on this, I just picked something

Reply:
I'll check chrome tonight.

Reply:
https://webrtc.googlesource.com/src/+/2e540a28c05104a4af4d44c978d59344e3cb53f7/modules/video_coding/nack_requester.cc#29

Reply:
Ok. But do they have a timer as well, so defacto they don't get all the way to 10?